http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -174,17 +169,12 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-      security:
```

Comment:
are these not needed?

Reply:
I think it inherits from the top level: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/swagger/api-docs.yaml#L22

Per-route security settings are only needed for overrides.

Looking on dev the swagger routes are authenticated correctly: https://leonardo.dsde-dev.broadinstitute.org/#/runtimes/listRuntimesByProject